include/gui/button.hpp

```diff
@@ -10,6 +10,15 @@
 class Button {
 public:
   static inline std::vector<Button*> g_buttons;
```

Comment:
Prefix public static inline variables with `g_`

Reply:
Resolved in 44e6aaf